index.html

```diff
@@ -12,16 +12,18 @@
     <header>
       <h1 id="logo">Aziz Mejri</h1>
       <button type="button" class="menu-bars">
-        <i class="fas fa-bars"></i>
+        <img id="menu-icon" class="menu-icon" src="menu.svg" alt="menu"/>
       </button>
-      <nav id="desktop-nav">
-        <ul>
-          <li><a class="nav-link" href="#projects">Portoflio</a></li>
-          <li><a class="nav-link" href="#about-me">About</a></li>
-          <li>
+      <nav class="main-nav" id="desktop-nav">
+        <img class="exit" src="exit.svg" alt="exit"/>
+        <ul class="ul-menu">
+          <li class="li-menu"><a class="nav-link" href="#projects">Portoflio</a> <div class="seprator"></div></li>
+          <li class="li-menu"><a class="nav-link" href="#about-me">About</a> <div class="seprator"></div></li>
+          <li class="li-menu">
```

Comment:
- [ ] Good job making the navbar links visible in the desktop version as requested by the reviewer :slightly_smiling_face: . However, it is not working, it seems that the header's background image impedes accessing those links (`Portfolio`, `About` and `Contact`). Kindly fix it to improve your project and follow the [Figma design.](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A838)

How your project looks like:

![image](https://user-images.githubusercontent.com/61767582/175439256-e0e2f8bc-8340-4bf8-82f0-bb6184a15b95.png)

How it should look like:
![image](https://user-images.githubusercontent.com/61767582/175439392-a1e9ddb0-2d80-4bd5-b0dd-61e35e1d31b9.png)

- [ ] [OPTIONAL] You did a great job improving the styling according to the code reviewer's request :tada: . In order to improve it, I suggest aligning the `About` nav link to match the Figma design. 

![1026effd-3486-4424-87e5-8ec5f9084e0a](https://user-images.githubusercontent.com/61767582/175440516-6f51e8ed-8b23-40d7-9ccf-86019646c214.jpg)